packages/client/hmi-client/src/types/Types.ts

```diff
@@ -160,6 +160,7 @@ export interface Dataset extends TerariumAsset {
     dataSourceDate?: Date;
     fileNames?: string[];
     datasetUrl?: string;
+    datasetUrls?: string[];
```

Comment:
Why do we need both?

Reply:
We don't, but I'm only about 80% sure we don't. `datasetUrl` is a holdover from old TDS. in `Dataset.java` I've added a comment with a link to a github issue to confirm this and remove after the eval - I dont want to commit that and have it blow up

Reply:
I agree that it is most probably a holdover, but my knowledge is less with respect to this.  And I love the idea that during the eval, maybe not the time to delete it :)

Reply:
yes hahah :D

on the java model:
TODO: IS THIS NEEDED? IS THIS FROM OLD TDS? https://github.com/DARPA-ASKEM/terarium/issues/3194